static/js/formatters-internal.js

```diff
@@ -599,7 +599,7 @@ export function getYoutubeUrl(videos = []) {
  * @returns a URL with text fragment URI component attached
  */
 export function getUrlWithTextHighlight(snippet, baseUrl) {
-  if (!isChrome()) {
+  if (!isChrome() || snippet.matchedSubstrings.length === 0) {
```

Comment:
do we also need to check whether snippet isn't undefined?

Reply:
as of now with documentsearch-directanswers card, I assumed snippet is always defined if the card is used. But yeah that should be check in case this could be use else where. will update!